skfuzzy/control/tests/test_controlsystem.py

```diff
@@ -53,7 +53,7 @@ def test_tipping_problem():
     tip_sim2.compute()
 
     assert tip_sim.output == tip_sim2.output
-    tst.assert_allclose(tip_sim.output['tip'], 20.244508118433625)
+    tst.assert_allclose(tip_sim.output['tip'], 19.847607362)
```

Comment:
When run with ridiculously finely sampled universes containing over a million points each,

``` python
food = Antecedent(np.linspace(0, 10, 1000001), 'quality')
service = Antecedent(np.linspace(0, 10, 1000001), 'service')
tip = Consequent(np.linspace(0, 25, 2000006), 'tip')
```

I see a result of 19.8578 using the previous method. So... bravo! I'm impressed.